py/objmodule.c

```diff
@@ -173,7 +175,11 @@ static const mp_module_delegation_entry_t mp_builtin_module_delegation_table[] =
 // Attempts to find (and initialise) a built-in, otherwise returns
 // MP_OBJ_NULL.
 mp_obj_t mp_module_get_builtin(qstr module_name, bool extensible) {
+    #if MICROPY_HAVE_REGISTERED_EXTENSIBLE_MODULES
     mp_map_elem_t *elem = mp_map_lookup((mp_map_t *)(extensible ? &mp_builtin_extensible_module_map : &mp_builtin_module_map), MP_OBJ_NEW_QSTR(module_name), MP_MAP_LOOKUP);
+    #else
+    mp_map_elem_t *elem = mp_map_lookup((mp_map_t *)&mp_builtin_module_map, MP_OBJ_NEW_QSTR(module_name), MP_MAP_LOOKUP);
+    #endif
```

Comment:
Instead of this, suggest:
```c
#if MICROPY_HAVE_REGISTERED_EXTENSIBLE_MODULES
mp_map_t *map = extensible ? &mp_builtin_extensible_module_map : &mp_builtin_module_map;
#else
mp_map_t *map = &mp_builtin_module_map;
#endif
mp_map_elem_t *elem = mp_map_lookup(map, MP_OBJ_NEW_QSTR(module_name), MP_MAP_LOOKUP);
```

Reply:
done!